test/DynamoCoreTests/PackageDependencyTests.cs

```diff
@@ -323,7 +364,14 @@ public void DependencyWithTypeLoads()
             // Assert ZTTestPackage is still a package dependency
             var packageDependencies = CurrentDynamoModel.CurrentWorkspace.NodeLibraryDependencies;
             Assert.Contains(new PackageDependencyInfo("ZTTestPackage", new Version("0.0.1")), packageDependencies);
-            Assert.IsTrue(packageDependencies.First().IsLoaded == false);
```

Comment:
so I totally get removing all these - BUT why not leave them so that the change you just made still has tests? Then we can remove the tests when we actually remove the property?

Reply:
Test both things is what I mean.

Reply:
Yes, checking both this properties now and we can remove them when removing the actual property.